config-model-api/src/main/java/com/yahoo/config/application/api/xml/DeploymentSpecXmlReader.java

```diff
@@ -247,11 +266,15 @@ private boolean trueOrMissing(String value) {
 
         String policy = upgradeElement.getAttribute("policy");
         switch (policy) {
-            case "canary" : return DeploymentSpec.UpgradePolicy.canary;
-            case "default" : return DeploymentSpec.UpgradePolicy.defaultPolicy;
-            case "conservative" : return DeploymentSpec.UpgradePolicy.conservative;
-            default : throw new IllegalArgumentException("Illegal upgrade policy '" + policy + "': " +
-                                                         "Must be one of " + Arrays.toString(DeploymentSpec.UpgradePolicy.values()));
+            case "canary":
```

Comment:
This reformatting just makes it harder to read ...

Reply:
Completely agree with you. I didn't notice this snuck in. IntelliJ loves doing this.

Reply:
Please don't autoformat ...

Reply:
Well, I never do on purpose. I don't even know what change may have caused IntelliJ to jump in here.

Reply:
Looks very much like auto-format was triggered somehow — doing it now does nothing 💡 
Any idea what may have caused it to trigger?

Reply:
No. I've turned that off.